middleware/context.go

```diff
@@ -18,12 +18,13 @@ func (k *contextKey) String() string {
 }
 
 var (
-	ctxKeyAccessKey     = &contextKey{"AccessKey"}
-	ctxKeyAccessQuota   = &contextKey{"AccessQuota"}
-	ctxKeyComputeUnits  = &contextKey{"ComputeUnits"}
-	ctxKeyRateLimitSkip = &contextKey{"RateLimitSkip"}
-	ctxKeyTime          = &contextKey{"Time"}
-	ctxKeyResult        = &contextKey{"Result"}
+	ctxKeyAccessKey        = &contextKey{"AccessKey"}
+	ctxKeyAccessQuota      = &contextKey{"AccessQuota"}
+	ctxKeyComputeUnits     = &contextKey{"ComputeUnits"}
+	ctxKeyRateLimitSkip    = &contextKey{"RateLimitSkip"}
+	ctxOverridePublicLimit = &contextKey{"OverridePublicLimit"}
```

Comment:
for consistency of naming... we have `ctxKeyRateLimitSkip` and you introduced `ctxOverridePublicLimit` ... please follow consistent conventions..

therefore, your name should be .. `ctxKeyRateLimitPublicOverride`